usr.sbin/sysinst/msg.mi.pt

```diff
@@ -0,0 +1,1528 @@
+ * modification, are permitted provided that the following conditions
+ * are met:
+ * 1. Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in the
+ *    documentation and/or other materials provided with the distribution.
+ * 3. The name of Piermont Information Systems Inc. may not be used to endorse
+ *    or promote products derived from this software without specific prior
+ *    written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY PIERMONT INFORMATION SYSTEMS INC. ``AS IS''
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL PIERMONT INFORMATION SYSTEMS INC. BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
+ * THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ */
+
+/* MI Message catalog -- portuguese, machine independent */
+
+/*
+ * We can not use non ascii characters in this message - it is displayed
+ * before the locale is set up!
+ */
+message sysinst_message_language
+{Mensagens de instala�ao em portug�s}
+
+message sysinst_message_locale
+{pt_BR.ISO8859-1}
+
+message	out_of_memory	{Out of memory!}
+message Yes {Sim}
+message No {Nao}
+message All {Todos}
+message Some {Alguns}
+message None {Nenhum}
+message none {nenhum}
+message OK {OK}
+message ok {ok}
+message On {Ativado}
+message Off {Desativado}
+message unchanged {Inalterado}
+message Delete {Apagar?}
+
+message install
+{instalar}
+
+message reinstall
+{reinstalar conjuntos para}
+
+message upgrade
+{atualizar}
+
+message hello
+{Esta ferramenta orientada por menu foi projetada para ajudar voc� a instalar o NetBSD em um
+disco r�gido ou atualizar um sistema NetBSD existente, com o m�nimo de trabalho.
+Nos menus a seguir, digite a letra de refer�ncia (a, b, c, ...) para
+selecionar um item ou digite CTRL+N/CTRL+P para selecionar o item seguinte/anterior.
+As teclas de seta e Page-up/Page-down tamb�m podem funcionar.
+Ative a sele�ao atual do menu digitando a tecla Enter.
+}
+
+message thanks
+{Obrigado por usar o NetBSD!
+}
+
+message installusure
+{Voc� escolheu instalar o NetBSD no seu disco r�gido. Isso mudar�
+as informa�oes no seu disco r�gido. Voc� deveria ter feito um backup completo
+antes deste procedimento! Este procedimento far� o seguinte:
+	a) Particionar seu disco
```

Comment:
Given that you're only saying "fará o seguinte", perhaps "Particionará" would make more sense?